app/javascript/components/get-version.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { Subscription } from 'react-apollo';
+import gql from 'graphql-tag';
+
+const getVersion = gql`
+  {
+    schemaSha
+  }
+`;
+
+let lastValue = null;
+
+export const Version = () => (
```

Comment:
I'm curious if we could call the ApolloClient directly with `subscribe` to skip React entirely for our reload logic:
https://www.apollographql.com/docs/react/api/core/ApolloClient/#ApolloClient.subscribe